client.go

```diff
@@ -31,6 +31,9 @@ var (
 	// APIHost is the base URL for Recurly API v3
 	APIHost = "https://v3.recurly.com"
 
+	// EuAPIHost is the base URL for Recurly API v3
+	EuAPIHost = "https://v3.eu.recurly.com"
```

Comment:
With Golang naming conventions, EU should be all caps. This gets a little awkward with `EUAPIHost`, so let's rename this to `APIHostEU`.

OR, let's change this to a map as that simplifies the base host lookup when we construct the client:

```
var (
    apiHosts = map[Region]string{
          US: "https://v3.recurly.com",
          EU: "https://v3.eu.recurly.com",
    }
)
```